### ISSUE TITLE
2.7.0 release

### ISSUE THREAD

**Issue by boubou19** [GTNewHorizons/GT-New-Horizons-Modpack#17229]
### Bouboi conditiones sine quibus non:
- [x] ~~Not a single recipe collision must be left. remaining: 365~~ some are left but nothing harmful
- [ ] ~~https://github.com/GTNewHorizons/GT5-Unofficial/blob/f8f995ef24813cc40b3454605f42d7ee06a46f51/src/main/java/bartworks/system/material/gtenhancement/PlatinumSludgeOverHaul.java#L736 must be removed once the recipes it patches are all edited in code~~ not possible without new material system or very dirty hacks.
- [ ] ~~https://github.com/GTNewHorizons/GT5-Unofficial/blob/f8f995ef24813cc40b3454605f42d7ee06a46f51/src/main/java/goodgenerator/loader/NaquadahReworkRecipeLoader.java#L644 must be removed once the recipes it patches are all edited in code~~ not possible without new material system or very dirty hacks.
- [x] https://github.com/GTNewHorizons/GT5-Unofficial/blob/f8f995ef24813cc40b3454605f42d7ee06a46f51/src/main/java/gtnhlanth/loader/BotRecipes.java#L240 once the recipes it patches are all edited in code
- [ ] ~~https://github.com/GTNewHorizons/GT5-Unofficial/blob/f8f995ef24813cc40b3454605f42d7ee06a46f51/src/main/java/gtnhlanth/loader/RecipeLoader.java#L4448 once the recipes it patches are all edited in code~~ not possible without new material system or very dirty hacks.
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17283
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17821
### Bigger issues that need fixing
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17832
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17242
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17251
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17258
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17087
- [x] https://github.com/GTNewHorizons/Dupes-Exploits-GTNH/issues/146
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17284
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17325
- [x] finalize PA deprecation https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17882#issuecomment-2486952360, https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17882#issuecomment-2486956504
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17401, https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17405
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17417
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17477#issuecomment-2378399480
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17523
- [x] a major problem with recipe checks, caching, and oredicts (all the same): https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17527, https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17510, https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17530
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17674, https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17607
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17442
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17655
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17794
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17788, https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17792
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17817
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17923
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/18081
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/18229
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17636

### Smaller issues that would also be nice to have fixed
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17446
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17188
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17159
- [ ] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17148
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/16969
- [x] clean up the remaining recipes with eut>practical voltage (see gt log with recipe debug on)
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17137
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/16530
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/15816, https://github.com/GTNewHorizons/Angelica/issues/383
- [x] https://github.com/GTNewHorizons/Angelica/issues/674, https://github.com/GTNewHorizons/Angelica/issues/572, https://github.com/GTNewHorizons/Angelica/issues/759
- [ ] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17261
- [ ] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17299
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17302
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17322
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17329
- [ ] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17330
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17335
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17390
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/14656
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17404
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17418
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17423
- [ ] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17444
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17440
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17477
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17517
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17518
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17522
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17593
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17592
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17583
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17620
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17610
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17623
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17507
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/16998
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17649
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17659
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17661
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17675
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17660
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17658
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17653
- [x] https://github.com/GTNewHorizons/Angelica/issues/412
- [ ] https://github.com/GTNewHorizons/Angelica/issues/665
- [ ] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/16413
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17705
- [ ] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17707
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17710
- [ ] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17685
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17748
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17752
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17786
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17841
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17864
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17825
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17347
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17876
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17910
- [ ] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17951, https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17662
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17994
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17498
- [ ] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/15949
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/18173
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/18226

### Potential issues that need testing/confirmation/details
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17246
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17219
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17163
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17079
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17135
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17281
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17267
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17287
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17340
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17378 
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17402
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17420
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17428
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17434
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17501
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17516
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17508
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/16658#issuecomment-2383142099
- [ ] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17586
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17581
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17578
- [ ] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17562
- [ ] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17615
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17613
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17652
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17663
- [ ] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17750, https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17262
- [ ] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17783
- [ ] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17935
- [x] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17949
- [ ] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/17997

**Comment by Dream-Master** [comment]
### Prs i like to add before 2.7.0 (Beta)

**Comment by Glease** [comment]
### MX-Random

- [ ] https://github.com/GTNewHorizons/MX-Random/pull/10/ I forget to remove draft status after great renaming. this one should be merged before 2.7.0 as LMA is now completely broken

**Comment by boubou19** [comment]
### PRs which should make it in Beta 4:
- [ ] https://github.com/GTNewHorizons/WitchingGadgets/pull/65
- [ ] https://github.com/GTNewHorizons/Hodgepodge/pull/438
- [ ] https://github.com/GTNewHorizons/Hodgepodge/pull/437
- [ ] https://github.com/GTNewHorizons/gendustry/pull/12
- [ ] https://github.com/GTNewHorizons/CodeChickenCore/pull/24
- [ ] https://github.com/GTNewHorizons/Thaumic_Exploration/pull/39
- [ ] https://github.com/GTNewHorizons/BlockRenderer6343/pull/22
- [ ] https://github.com/GTNewHorizons/GT5-Unofficial/pull/3467
- [ ] https://github.com/GTNewHorizons/NewHorizonsCoreMod/pull/1075
- [ ] https://github.com/GTNewHorizons/AE2FluidCraft-Rework/pull/237
- [ ] https://github.com/GTNewHorizons/ModularUI/pull/86
- [ ] https://github.com/GTNewHorizons/AE2FluidCraft-Rework/pull/236
- [ ] https://github.com/GTNewHorizons/Crops-plus-plus/pull/81
- [ ] https://github.com/GTNewHorizons/LunatriusCore/pull/8
- [ ] https://github.com/GTNewHorizons/EnderIO/pull/178
- [ ] https://github.com/GTNewHorizons/GT5-Unofficial/pull/3432
- [ ] https://github.com/GTNewHorizons/NewHorizonsCoreMod/pull/1052
- [ ] https://github.com/GTNewHorizons/GT5-Unofficial/pull/3378
- [ ] https://github.com/GTNewHorizons/NewHorizonsCoreMod/pull/1042
- [ ] https://github.com/GTNewHorizons/Binnie/pull/54
- [ ] https://github.com/GTNewHorizons/GT5-Unofficial/pull/3305 (not sure)
- [ ] https://github.com/GTNewHorizons/Binnie/pull/53

**Comment by boubou19** [comment]
Things to deal with before RC1:
- [ ] remove PA controller recipe
- [ ] https://github.com/GTNewHorizons/GT5-Unofficial/pull/3467
- [ ] https://github.com/GTNewHorizons/BlockRenderer6343/pull/22

**Comment by boubou19** [comment]
things to deal with before stable:

high prio:
- [ ] https://github.com/GTNewHorizons/NotEnoughItems/pull/565
- [ ] https://github.com/GTNewHorizons/BloodArsenal/pull/25
- [ ] https://github.com/GTNewHorizons/CarpentersBlocks/pull/12

low prio:
- [ ] https://github.com/GTNewHorizons/GT5-Unofficial/pull/3595
- [ ] https://github.com/GTNewHorizons/Botania/pull/64

investigations required on these:
- [ ] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/18239
- [ ] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/18238
